examples/get_iss_data.py

```diff
@@ -1,13 +1,17 @@
 from __future__ import division
 
+import argparse
 import io
+import os
 import json
 import zipfile
 
-import click
 import requests
+from slicedimage import ImageFormat, ImagePartition, Tile, Writer
```

Comment:
What does `lims` stand for?

Reply:
It determines what values the minimum and maximum value of the (usually linear) colormap map onto in the data. It's to prevent cases where a very large high value outlier compresses the colorspace by, for example, assigning 99% of the values to something that looks more or less like "black" in a greyscale colormap.